industry_real_estate/data/website_controller_page.xml

```diff
@@ -104,7 +104,6 @@
         <field name="page_type">listing</field>
         <field name="model_id" ref="analytic.model_account_analytic_account"/>
         <field name="website_id" ref="website.default_website"/>
-        <field name="website_published" eval="True"/>
```

Comment:
This page should stay published. You'll need to update the test to allow this one.